intera_interface/src/intera_motion_interface/random_walk.py

```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env python
+
+# Copyright (c) 2016-2017, Rethink Robotics Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from copy import deepcopy
+import random
+import rospy
+
+class RandomWalk(object):
+    """
+    This class is used to create a random walk in a bounded state space
+    """
+
+    def __init__(self):
+        """
+        Constructor - creates an empty random walk object. Be sure to set the
+        lower and upper limits, as well as the past point before calling.
+        """
```

Comment:
This comment is confusing. How can we call methods before calling the constructor? I think that it should be: "be sure to set lower and upper limits before attempting to call getNextPoint()".